.changeset/early-grapes-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/ai-robots-txt': major
```

Comment:
`major` is also fine, but I think `minor` is better. It may be early to make it as version `1.x`.